R/app.R

```diff
@@ -120,7 +120,7 @@
 #' to \code{true}, or a timeout happens. It returns \code{TRUE} is the
 #' expression evaluated to \code{true}, possible after some waiting.
 #'
-#' \code{find_widget()} finds the corresponding HTML element of a Shiny
+#' \code{app$find_widget()} finds the corresponding HTML element of a Shiny
 #' widget. It returns a \code{\link{widget}} object.
 #'
 #' \code{expect_update()} is one of the main functions to test Shiny apps.
```

Comment:
They are exactly the same. Basically you can write it both ways: `app$expect_update()` or `expect_update(app, ...)`. I cannot decide which one is more natural. Yes, it is a method, so `app$expect_update()` makes sense. But I also want people to think about it as a regular `testthat` expectation, and then a non-method `expect_update()` is more natural. 

What do you think?